agent/cmd/start.go

```diff
@@ -40,8 +40,11 @@ const (
 	flagDeregistrationHandler = "deregistration-handler"
 	flagEnvironment           = "environment"
 	flagExtendedAttributes    = "custom-attributes"
+	flagFlushInterval         = "flush-interval"
```

Comment:
Should we also prefix this flag with `metrics`? Or maybe use `statsd` for all related flags? So it's easier to see that it's related to statsd using the flag name only!

Reply:
Yeah, I'm honestly thinking we should prefix all things statsd related with `statsd`! I noticed the naming convention was somewhat inconsistent.